src/main/java/duke/main/Parser.java

```diff
@@ -0,0 +1,177 @@
+package duke.main;
+
+import duke.command.AddDeadlineCommand;
+import duke.command.AddEventCommand;
+import duke.command.AddTodoCommand;
+import duke.command.Command;
+import duke.command.DeleteCommand;
+import duke.command.DoneCommand;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.exception.DukeException;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+class Parser {
+    static Command parse(String fullCommand) throws DukeException {
+        if (isExitCommand(fullCommand)) {
```

Comment:
In my opinion, this could be better expressed as a `switch` statement. Maybe you could add some `String` or `enum` identifier to each of your command classes to facilitate this.